main.py

```diff
@@ -55,10 +55,27 @@ def write_cache(live_cache: dict):
     with open("roles.json", "w") as f:
         json.dump(live_cache, f, indent=2)
 
+#Below is the code for sending ZOOM LINK by Abir
```

Comment:
How have you tested this function? It looks like most code will never be reached?

At the  start, we get the day. Let's imagine today is Friday. "now" should be equal to 4.

Therefore...
```python3
if now == 2: (False)
    ...skips everything
```

Reply:
I thought the zoom link is supposed to be sent every week only in Wednesdays. As the club meetings are only on Wednesdays. Thus, if now = 2 (True), the zoom link will be sent out to the channel on every Wednesday before the meeting.